mappings/net/minecraft/client/gui/screen/Screen.mapping

```diff
@@ -42,6 +43,10 @@ CLASS net/minecraft/unmapped/C_wrmtlwqx net/minecraft/client/gui/screen/Screen
 		ARG 1 client
 		ARG 2 width
 		ARG 3 height
+	METHOD m_efvbcpbf preInit ()V
+		COMMENT Called before the {@linkplain net.minecraft.client.gui.screen.Screen#init() init} method.
+		COMMENT
+		COMMENT <p>This method will not be called when the screen gets resized.</p>
```

Comment:
```suggestion
		COMMENT <p>
		COMMENT This method will not be called when the screen gets resized.</p>
```

Reply:
oh, there's a `</p>` there too

Reply:
I hope I addressed all of the mentioned things, I'm sorry for the inconvenience. Regarding the `</p>` (it's already gone), it should maybe be clarified in the guidelines that closing tags should be discarded (or at least I couldn't find where it says that).